ncm-opennebula/src/main/perl/opennebula.pm

```diff
@@ -105,7 +105,7 @@ sub create_or_update_something
         $new = $one->$cmethod("$template");
```

Comment:
@stdweird fixed, and sorry for the noise we don't need a new `NEt::OpenNebula` version